workflow/workflow-demos/src_hd/workflow/businessdata/BusinessDataDossierLazyBrowser/BusinessDataDossierLazyBrowser.xhtml

```diff
@@ -29,10 +29,9 @@
         paginatorPosition="bottom" rows="5"
         paginatorTemplate="{RowsPerPageDropdown} {FirstPageLink} {PreviousPageLink} {CurrentPageReport} {NextPageLink} {LastPageLink}"
         currentPageReportTemplate="{startRecord} - {endRecord} of {totalRecords}"
-        rowsPerPageTemplate="5,10,15,25,50,100" lazy="true"
-        sortBy="#{dossier.name.ascii}">
+        rowsPerPageTemplate="5,10,15,25,50,100" lazy="true">
         <p:column headerText="Dossier Name"
-          sortBy="#{dossier.name.ascii}">
+          sortBy="#{dossier.name.ascii}" sortOrder="ASCENDING">
```

Comment:
according to the primefaces online help possible values for sortOrder are "asc" or "desc".
But ASCENDING, DESCENDING also work. so it is ok.